UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService.UnitTests/Services/ConfigurationServiceTests.cs

```diff
@@ -476,6 +548,7 @@ private List<BessConfig> GetFakeConfigurationSettingNotEnabled()
         }
 
         #region GetSalesCatalogueDataProductResponse
```

Comment:
Add a comment instead of a region for method

Reply:
Shall I remove it completely instead of comment?